clients/python/test/e2e/data/server_client_compatibility.json

```diff
@@ -6,45 +6,70 @@
   },
   "1": {
     "server": "api.osparc-master.speag.com",
-    "client": "production",
+    "client": "staging",
```

Comment:
cool! so i guess staging client will be able to handle all the server API with DEV_FEATURES=1, right?

Reply:
🤔 according to https://github.com/ITISFoundation/osparc-simcore/tree/master/services/api-server#backwards-compatibility-of-the-server-api `staging` in the client will be compatible with server `staging` and `production`. So only `master` on the client side will be compatible with `DEV_FEATURES=1`